2.1/builder/caddy-builder.sh

```diff
@@ -6,4 +6,4 @@ for p; do
 	args="$args --with $p"
 done
 
-/usr/bin/xcaddy build $CADDY_SOURCE_VERSION --output /usr/bin/caddy $args
\ No newline at end of file
+/usr/bin/xcaddy build --output /usr/bin/caddy $args
```

Comment:
I don't think this is right, because it will make it build from the latest published tag instead of the same one as the builder image is tagged for 🤔

If users want the latest tag, then they can use `xcaddy` directly instead of this shell script.

Reply:
See https://github.com/caddyserver/caddy-docker/pull/113#discussion_r487547252